Client/Cliqz/Frontend/Resources/fr.lproj/Cliqz.strings

```diff
@@ -1,144 +1,144 @@
 /* The text shown in the search field */
-"Search" = "Search";
+"Search" = "Rechercher";
 
 /* Error message title */
-"Sorry" = "Sorry";
+"Sorry" = "Désolé";
 
 /* Error message when JS scripts for search can't be loaded */
-"Something went wrong, the script wasn't loaded." = "Something went wrong, the script wasn't loaded.";
+"Something went wrong, the script wasn't loaded." = "Quelque chose c'est mal passé, le script n'a pas pu être chargé.";
 
 /* Button label to cancel entering a URL or search query
  Cancel */
-"Cancel" = "Cancel";
+"Cancel" = "Annuler";
 
 /* Button label on Alert view */
-"Retry" = "Retry";
+"Retry" = "Réessayer";
 
 /* Search Recommendations and top visited sites title */
-"Search recommendations" = "Recommendations";
+"Search recommendations" = "Recommandés";
 
 /* Search history title */
-"Search history" = "History";
+"Search history" = "Historique";
 
 /* Show Cliqz legal page. See https://cliqz.com/legal" */
-"Imprint" = "Imprint";
+"Imprint" = "Légal";
 
 /* Menu item in settings used to open FAQs & Support cliqz url where people can submit feedback */
 "FAQs & Support" = "FAQs & Support";
 
 /* Open search section of settings */
-"Search" = "Search";
+"Search" = "Moteurs de recherche";
 
 /* Footnote message for feedback */
-"Feedback to Cliqz Browser (Version %@) for iOS (Version %@) from %@" = "Feedback to Cliqz Browser (Version %@) for iOS (Version %@) from %@";
+"Feedback to Cliqz Browser (Version %@) for iOS (Version %@) from %@" = "Feedback pour le navigateur Cliqz (Version %@) de iOS (Version %@) de %@";
 
 /* Title for No Email account pop-up */
-"No Email account title" = "No Email account";
+"No Email account title" = "Aucune adresse email";
 
 /* Text for the missing email account for sending feedback pop-up */
-"No Email account message" = "Please setup an Email account to be able to send a feedback.";
+"No Email account message" = "Veuillez entrer une adresse email afin d'envoyer un feedback.";
 
 /* Error title */
-"The Internet connection appears to be offline." = "The internet connection appears to be offline.";
+"The Internet connection appears to be offline." = "La connexion internet semble interrompue.";
 
 /* Title for `Top news` home shortcut. */
-"Top news" = "Top news";
+"Top news" = "Le meilleur de l'actualité";
```

Comment:
I think that we haven't translated it for Desktop, and I even can't find where it is use